tpm/aziot-tpm-sys/README.md

```diff
@@ -0,0 +1,36 @@
+# aziot-tpm-sys crate
+
+This crate is the unsafe C to Rust interface for the TPM API library.
+
+This crate represents the functions that the TPM API implements. This crate is
+used by the `aziot-tpm-rs` crate to provide more Rust-friendly interfaces.
+
+## TPM functionality
+
+You may need additional setup for a TPM device see the in-tree dev docs for
+more details.
+
+## Memory allocation
+
+The current TPM API functions expect the calling function to allocate memory for
+the the caller to use. The caller (in this case, the Rust crate) is expected to
+free this memory.
+
+## Build dependencies
```

Comment:
Just leave this section out. Apart from being incomplete ("On Debian based linux systems"), it's redundant with `install-build-deps.sh` and would have to be kept in sync.